_posts/blog/2020-03-16-chi-hack-night-going-remote.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: blog
+categories: 
+  - blog
+title: "Introducing Remote Chi Hack Nights"
+description: "Are you already feeling that Tuesday night void? Looking for a way to help out and contribute during the COVID-19 outbreak? Chi Hack Nights are going remote! Same time, but now from the comfort of your own home / quarantine location!"
+date: 2020-03-16
+image: 
+author: Eric Sherman
+author_url:
+author_image: /images/people/eric_sherman.jpg
+author_bio: "Board Member, Chi Hack Night"
+author2: Derek Eder
+author2_url: https://twitter.com/derekeder
+author2_image: /images/people/derek_eder.jpg
+author2_bio: "President, Chi Hack Night"
+
+published: true
+---
+
+*Due to the ongoing outbreak of COVID-19 (Coronavirus), in-person [Chi Hack Night is Postponed Until Further Notice](/blog/2020/03/09/chi-hack-night-postponed.html)*
```

Comment:
Does this need to be here? Seems to detract from the opening paragraph.

Reply:
It doesn't have to be there like that, but without something, the post doesn't actually say anything about there being no in person hack night.

Reply:
@derekeder what do you think? We could also edit something below, like 

``` Are you feeling that Tuesday night void since [in person Chi Hack Nights got postponed](/blog/2020/03/09/chi-hack-night-postponed.html)? ```